modules/ROOT/pages/managing-users.adoc

```diff
@@ -3,31 +3,85 @@ ifndef::env-site,env-github[]
 
 When the registration is approved, the user is removed from the list on the *Pending registration requests* page and added to the list of members on the *Portal members* page. 
 
 If the registration is rejected, the user is removed from the list and can't log in and access the portal.
 
 === Approve or Reject Member Registration
-After a guest user registers for an account using self-registration, approve or reject the request. 
+After a guest user registers for an account using self-registration, approve or reject the requests and assign the user to user groups. All users who self-register are assigned as members. You can assign member users to multiple user groups.
 
 . Click *Pending registration requests*.
 . To find a user, enter the user's name or email in the *Search by name or email* box.
+. To assign the user to a user group:
+.. Click *Members* and select one or more user groups.
+.. Click *Apply*.
 . Click *Approve* or *Reject*.
 
+== Manage User Groups
+Manage user groups using the *User groups* page to create and manage a group of users who can see specific APIs in your portal.   
+
+=== Create User Groups
+Create groups of users who can see specific APIs in the portal.
+
+. From the *User groups* page, click *Create user group*.
+. Enter a name for the group and an optional description.
+. Click *Save*.
+
+
+=== Delete User Groups
+Delete user groups you don't need. Before deleting a user group, remove all users,  APIs, and the visibility for all API instances assigned to the user group. 
+
+To delete a user group, from the *User groups* page, select a user group and click *Delete user group*.
+
+=== Map User Groups to External Identity Providers
+For users of a user group to log in to your portal using SSO, map the user group to an existing external identity provider. To avoid login issues, ensure you use the same group name and identity provider name configured for the identity provider. 
+
+To add the user group to the corresponding profile in API Experience Hub:
+
+. From the *User groups* page, select a group.
+. In the *External IdP groups* tab, click *+ Add group mapping*.
+. Enter the group name and select an Salesforce identity provider name.
+. Click *Save*.
+
+
+
+
+=== View APIs That User Groups Can See
+View the APIs that are visible by a user group for portal members and also the APIs assigned only to a user group. The *APIs visible to the user group* tab shows all of the APIs that members of the portal can see.  
+
+. From the *User groups* page, select a user group.
+. Select the *APIs visible to the user group* tab.
+. To see APIs assigned to the group only, move the *Show only APIs assigned to this group* slider to on.
```

Comment:
See the [SF guidelines](https://confluence.internal.salesforce.com/x/RQWGC) for how to handle toggles.